app/services/course_creation_step_service.rb

```diff
@@ -18,7 +18,27 @@ def get_previous_step(steps, current_step)
     steps[previous_step_index]
   end
 
-  def get_workflow_steps(course)
+  def get_workflow_steps(course, provider)
+    if provider.recruitment_cycle_year.to_i >= Provider::CHANGES_INTRODUCED_IN_2022_CYCLE
```

Comment:
a bit nitty but can we just use `course.provider.recruitment_cycle.year` here so we don't have to pass the provider in.

Reply:
Not nitty at all, think it's the sensible thing to do here. Have just refactored 👍